### DISCUSSION TITLE
Package queries don't work with spaces in the value

### DISCUSSION THREAD

**Discussion by richardtallent** [ckan/ckan#6558] [Q&A] [answered]
**CKAN version**

Unknown version. We're trying to query the open.alberta.ca site (it's not our site).

**Describe the bug**

Queries on the "pubtype" attribute that include a space are not working. For example, this search:

https://open.alberta.ca/api/3/action/package_search?q=pubtype:Orders%20and%20Directives&rows=50

Should return results including this one:

https://open.alberta.ca/api/3/action/package_show?id=9781460141571

The same issue exhibits for "Legislation and Regulations", so it definitely appears to be a space issue.

The documentation here suggests the normal "%20" escaping should work:
 
https://docs.ckan.org/en/ckan-2.7.3/api/

**Steps to reproduce**

See above.

**Expected behavior**

See above.

**Additional details**

n/a

**Comment by amercader** [comment, marked as answer]
If your query terms contain spaces you need to quote the value with `"`:

https://open.alberta.ca/api/3/action/package_search?q=pubtype:%22Orders%20and%20Directives%22&rows=50

Are these the results you expect?

**Comment by richardtallent** [comment]
Yes, that solves the issue! Thanks!
